installer/manifests/keptn/charts/control-plane/templates/webhook-service.yaml

```diff
@@ -124,6 +124,8 @@ data:
     kubernetes.default
     kubernetes.default.svc
     kubernetes.default.svc.cluster.local
+    svc.cluster.local
+    cluster.local
```

Comment:
doesn't this mean that you can't send a webhook to anything running on k8s now?
So all webhook traffic needs to leave the cluster and east-to-west traffic is basically forbidden, am I missing something?

Reply:
arg that was for testing I revert it

Reply:
having `svc.cluster.local` still has similar behaviour...

Reply:
so at the end we discussed to go that route, and disable any connection by default, I revert the changes to include both denied